### ISSUE TITLE
Npm installed 2.4.1 does not contain the src folder - hence the index.d.ts is not available

### ISSUE THREAD

**Issue by frjtrifork** [launchdarkly/js-client#109]
If I do `npm install ldclient-js@2.4.1` and then` ls -la node_modules/ldclient-js/` there is no src/ folder - but src is referenced in package.json:
```
  "types": "./src/index.d.ts",
  "version": "2.4.1"
```
Which means that our typescript builds fail when we reference LDClient in 2.4.1. 

If I do `npm install ldclient-js@2.4.0` then `ls -la node_modules/ldclient-js/` does have a src/ folder with the index.d.ts file in it, so our Typescript imports work fine with 2.4.0.

So I guess the npmjs publish now ignores the src folder? 
If that is the case it would be very nice if you could move the index.d.ts to the dist, or somewhere else that is included in the publish and update the types property in package.json.

**Comment by frjtrifork** [comment]
```
lerna ERR! > react-scripts-ts build
lerna ERR!
lerna ERR! Creating an optimized production build...
lerna ERR! Starting type checking and linting service...
lerna ERR! Using 1 worker with 2048MB memory limit
lerna ERR! ts-loader: Using typescript@2.9.2 and /Users/frj/dev/project/Project/tsconfig.prod.json
lerna ERR! Failed to compile.
lerna ERR!
lerna ERR! /Users/frj/dev/project/Project/packages/feature-flagging/build/LDClientWrapper.d.ts
lerna ERR! (1,26): Could not find a declaration file for module 'ldclient-js'. '/Users/frj/dev/project/Project/packages/feature-flagging/node_modules/ldclient-js/dist/ldclient.cjs.js' implicitly has an 'any' type.
lerna ERR!   Try `npm install @types/ldclient-js` if it exists or add a new declaration (.d.ts) file containing `declare module 'ldclient-js';`
```

**Comment by apucacao** [comment]
Hi @frjtrifork,

Sorry for the inconvenience. That's a mistake on our part -- we'll make sure that the definition is part of the npm package again.

**Comment by eli-darkly** [comment]
This has been fixed in the 2.5.0 release.
